ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -251,13 +275,14 @@ export const WorkspaceCard = fp.flow(withNavigation)(
               >
                 <FlexColumn style={{ marginBottom: 'auto' }}>
                   <FlexRow style={{ alignItems: 'flex-start' }}>
-                    <Clickable
-                      style={{
-                        cursor: tierAccessDisabled ? 'not-allowed' : 'pointer',
-                        ...styles,
-                      }}
+                    <a
```

Comment:
@NehaBroad  as of very recently I'm noticing clicking a workspace card results in a full page reload. I suspect this line is the problem - using `<StyledRouterLink>` instead of `<a>` here will probably resolve it. Otherwise this is just a normal link, which will do a full page navigation when you click it; react router does not intercept regular `<a>` interactions.